### PR TITLE
OSL tweaks, texture, luminance, labels

### PR DESCRIPTION
**Pull request by luisbarrancos**
This should fix the parameter name changes. Only the labels were changed, but it'll be a good idea to be thorough to make sure we're on the same wavelenght here (pun intended).
The luminance function was updated, the texture fixes incorporated again.

### CODE REVIEW

File: src/appleseed.shaders/src/appleseed/as_standard_surface.osl [hunk 1]
```diff
@@ -97,14 +97,28 @@ shader as_standard_surface
         int as_maya_attribute_hidden = 1,
         int gafferNoduleLayoutVisible = 0
     ]],
+    int in_subsurface_profile = 0
```
Comment (dictoon) [inline src/appleseed.shaders/src/appleseed/as_standard_surface.osl]: Does that mean the default is now Gaussian, whereas it was Normalized Diffusion before? If so, we need to change it back to Normalized Diffusion (2).

File: src/appleseed.shaders/src/appleseed/as_standard_surface.osl [hunk 2]
```diff
@@ -298,7 +312,7 @@ shader as_standard_surface
         float max = 1.0,
         string label = "Coating Reflectivity",
         string page = "Coating",
-        string help = "Coating specular amount."
+        string help = "Coating specular reflectivity."
```
Comment (dictoon) [inline src/appleseed.shaders/src/appleseed/as_standard_surface.osl]: A very minor thing, but some help strings end with a dot, some others not.

Comment (luisbarrancos) [reply src/appleseed.shaders/src/appleseed/as_standard_surface.osl]: You're right, now it's as good a time to fix this as any other. I'll update the remaining nodes affected by this as well.

File: src/appleseed.shaders/src/appleseed/as_standard_surface.osl (hunk 1 above)
Comment (luisbarrancos) [reply src/appleseed.shaders/src/appleseed/as_standard_surface.osl]: Ok.

**Pull request merged**